docs/src/inverse_problems/structural_identifiability.md

```diff
@@ -0,0 +1,168 @@
+
+## Global identifiability analysis
+
+### Basic example
+
+Local identifiability can be assessed using the `assess_identifiability` function. For each model quantity (parameters and variables), it will asses whether they are:
+- globally identifiable.
+- locally identifiable.
+- Unidentifiable.
+  
+To it, we provide our `ReactionSystem` model and a list of quantities that we are able to measure. Here, we consider a Goodwind oscillator (a simple 3-component model, where the three species $M$, $E$, and $P$ are produced and degraded, which may exhibit oscillations)[^2]. Let us say that we are able to measure the concentration of $M$, we then provide designate this using the `measured_quantities` argument. We can now assess identifiability in the following way:
+```example si1
+using Catalyst, Logging, StructuralIdentifiability
+goodwind_oscillator = @reaction_network begin
+    (pₘ/(1+P), dₘ), 0 <--> M
+    (pₑ*M,dₑ), 0 <--> E
+    (pₚ*E,dₚ), 0 <--> P
+end
+assess_identifiability(goodwind_oscillator; measured_quantities=[:M], loglevel=Logging.Error)
+```
+From the output, we find that `E(t)`, `pₑ`, and `pₚ` (the trajectory of $E$, and the production rates of $E$ and $P$, respectively) are non-identifiable. Next, `dₑ` and `dₚ` (the degradation rates of $E$ and $P$, respectively) are locally identifiable. Finally, `P(t)`, `M(t)`, `pₘ`, and `dₘ` (the trajectories of `P` and `M`, and the production and degradation rate of `M`, respectively) are all globally identifiable. We note that we also imported the Logging package, and provided the `loglevel=Logging.Error` input argument StructuralIdentifiability functions generally provides a large number of output logging messages. Hence, we will use this argument (which requires the Logging package) throughout this tutorial.
+
+
+Next, we can assess identifiability in the case where we can measure all three species concentrations:
+```example si1
+assess_identifiability(goodwind_oscillator; measured_quantities=[:M, :P, :E], loglevel=Logging.Error)
+```
+in which case all initial conditions and parameters become identifiable.
+
+
+### Indicating known parameters
+In the previous case we assumed that all parameters are unknown, however, this is not necessarily true. If there are parameters which value's are known, we can supply these using the `known_p` argument. Indeed, this might turn other, previously unidentifiable, parameters identifiable. Let us consider the previous example, where we measure the concentration of $M$ only, but also happen to know the production rate of $E$ ($pₑ$):
+```example si1
+assess_identifiability(gwo; measured_quantities=[:M], known_p=[:pₑ], loglevel=Logging.Error)
+```
+Not only does this turn the previously non-identifiable `pₑ` (globally) identifiable (which is obvious, given that its value is now known), but this additional information improve identifiability for several other network components.
```

Comment:
```suggestion
Not only does this turn the previously non-identifiable `pₑ` (globally) identifiable (which is obvious, given that its value is now known), but this additional information allows identifiability for several other network components.
```

Reply:
Since we might go from:

unidentifiable -> locally identifiable
unidentifiable -> globally identifiable
locally identifiable -> globally identifiable

and it is actually somewhat nuanced (and not just identifiable or not), still sure "allows" is correct?
(I'm happy to have it, just checking whether you accounted for this)

Reply:
The code prints the identifiability for all components, so the reader can compare to the previous case (where `pₑ` is not known) and see how things changes. Do you think this is enough to make things make sense?